keyboards/deltasplit75/pro_micro.h

```diff
@@ -0,0 +1,362 @@
+/*

```

Comment:
This file shouldn't be needed anymore - we have `pro_micro.h` in the `/drivers/avr` folder now ;)